File: src/main/resources/com/teamfour/iae/extracteds/20200602036/main.py
import sys

# Bubble sort algorithm
def bubble_sort(arr):
    n = len(arr)
    for i in range(n):
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]

# Get the list of strings from command line arguments
str_list = sys.argv[1:]

# Sort the list of strings using bubble sort
bubble_sort(str_list)

# Print the sorted list of strings
print("Sorted list of strings:")
for s in str_list:
    print(s)

File: src/main/resources/com/teamfour/iae/extracteds/20200602036/main.java
public class main {
    // Bubble sort algorithm
    public static void bubbleSort(String[] arr) {
        int n = arr.length;
        for (int i = 0; i < n - 1; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                if (arr[j].compareTo(arr[j+1]) > 0) {
                    // swap arr[j] and arr[j+1]
                    String temp = arr[j];
                    arr[j] = arr[j+1];
                    arr[j+1] = temp;
                }
            }
        }
    }

    public static void main(String[] args) {
        // Get the list of strings from command line arguments
        String[] strArray = args;

        // Sort the list of strings using bubble sort
        bubbleSort(strArray);

        // Print the sorted list of strings
        System.out.println("Sorted list of strings:");
        for (String str : strArray) {
            System.out.println(str);
        }
    }
}
